packages/animations/lib/src/fade_scale_transition.dart

```diff
@@ -135,6 +139,19 @@ class FadeScaleTransition extends StatelessWidget {
   ///    when it is used as a page transition.
   final Animation<double> animation;
 
+  /// The alignment of scale anchor point, relative to it's child.
+  ///
+  /// By default, the child scales from it's center when fading in. In a
+  /// variation on the default behavior, the scale anchor point can be
+  /// repositioned from the center of a child to another location within
+  /// the child.
+  ///
+  /// For example, when an overflow icon (the parent component) triggers a menu
+  /// (the child component), the menu's anchor point can be positioned at the
+  /// top right corner of the menu so that it appears to scale from the overflow
+  /// icon during a transition.
```

Comment:
```dart
/// To modify alignment, `FadeScaleTransitionConfiguration` will 
/// need to be extended and this new configuration class will need to be used instead:
/// ```
/// class _ModifiedFadeScaleTransitionConfiguration extends FadeScaleTransitionConfiguration {
///   const _ModifiedFadeScaleTransitionConfiguration();
///
///   @override
///   Widget transitionBuilder(
///     BuildContext context,
///     Animation<double> animation,
///     Animation<double> secondaryAnimation, 
///     Widget child,
///   ) {
///     return FadeScaleTransition(
///       animation: animation,
///       alignment: Alignment.topRight, // add this line
///       child: child,
///     );
///   }
/// }
/// ```
```

Reply:
I'll add this doc to `FadeScaleTransitionConfiguration` class instead.